grunt_tasks/webpack.config.js

```diff
@@ -25,7 +25,6 @@ var webpack = require('webpack');
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
 
 var paths = require('./webpack.paths.js');
-var es2015Preset = require.resolve('babel-preset-es2015');
```

Comment:
Thanks, I remember seeing some discussion about that previously, but never read the details myself. I'll retest with the current config, and switch back to `require.resolve` if it's still broken.

I think we really need to ensure that there are code comments when these sorts of non-obvious workarounds are implemented.